content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,146 @@ schema: olm.channel
 
 Here, a channel is generated for each template channel which differs by minor version, and each channel has a `replaces` edge from the predecessor channel to the next-lesser minor bundle version. Please note that at no time do we transgress across major-version boundaries with the channels, to be consistent with [the semver convention](https://semver.org/) for major versions, where the purpose is to make incompatible API changes.
 
+## Composite Template
+A `composite template` can help an operator author manage FBC contributions to multiple catalogs. The template functionality is composed of schemas which represent the author's role and the catalog maintainer's role, and rendering the template performs an explicit negotiation between them.
+
+### Usage
+```sh
+opm alpha render-template composite [flags]
+```
+
+| Flag                | Description                                                                            |
+| ------------------- | -------------------------------------------------------------------------------------- |
+| -f, --catalog-config string | File to use as the catalog configuration file (default "catalogs.yaml")        |
+| -c, --composite-config string | File to use as the composite configuration file (default "catalog/config.yaml") |
+| -h, --help          | help for composite                                                                     |
+| -o, --output string | Output format (json|yaml) (default "json")                                             |
+| --validate          | whether or not the created FBC should be validated (i.e 'opm validate') (default true) |
+| --skip-tls-verify   | skip TLS certificate verification for container image registries while pulling bundles |
+| --use-http          | use plain HTTP for container image registries while pulling bundles                    |
+
+
+
+
+### Specifications
+The `composite template` is composed of two schemas that represent the operator author's role and the catalog maintainer's role.
+
+#### Operator Author
```

Comment:
Unfortunately, this h4 anchor is skipped by the current docsy config.  I was curious to see what would happen if we changed ./config.toml
```diff
diff --git a/config.toml b/config.toml
index 0b6a077..c026d18 100644
--- a/config.toml
+++ b/config.toml
@@ -34,6 +34,8 @@ pygmentsStyle = "tango"
   [markup.goldmark]
     [markup.goldmark.renderer]
       unsafe = true
+  [markup.tableOfContents]
+    endLevel = 4

 # Image processing configuration.
 [imaging]
 ```
 
 I have some concerns that it might yield too much noise in the right-hand TOC, but it would be nice to allow folks to dial in to this level, or maybe we want to find another way to structure this for similar effect.

Reply:
https://github.com/operator-framework/olm-docs/pull/286/commits/8c89bd367d6835faed14561c8660ad7acb18e3fc